src/status_im2/contexts/quo_preview/preview.cljs

```diff
@@ -199,6 +219,7 @@
        (case (:type desc)
          :boolean [customizer-boolean descriptor]
          :text    [customizer-text descriptor]
+         :number  [customizer-number descriptor]
          :select  [customizer-select descriptor]
```

Comment:
Nitpick: Why not `type-customizer` for the name :)

Reply:
@ibrkhalil, if I could, I would rewrite the quo preview entirely, not because it's bad, but for lots of small things. But given that we're exploring Storybook now, I think it's probably not worth our time.

So why not `type-customizer`? Because I just wanted to follow the convention already in use here `customizer-XYZ`.